apps/analytics/vercel.json

```diff
@@ -16,6 +12,10 @@
 				{ "key": "Access-Control-Allow-Methods", "value": "GET" },
 				{ "key": "Cache-Control", "value": "public, max-age=31536000, immutable" }
```

Comment:
this cache-control header seems really aggressive for an ingestion endpoint... is this definitely what we want?